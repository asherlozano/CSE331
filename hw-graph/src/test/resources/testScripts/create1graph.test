# Create graph test file
# Test that a graph can be created
# A graph can be connected to nodes and labeled edges


# Create a graph
CreateGraph  g1

# Print the graphs nodes
ListNodes g1