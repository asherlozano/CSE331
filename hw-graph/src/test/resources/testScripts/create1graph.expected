# Create graph expected file
# Test that a graph can be created
# A graph can be connected to nodes and labeled edges

# Create a graph
created graph g1

# Print the nodes in the graph
g1 contains: